contrib/python/src/python/pants/contrib/python/checks/checker/print_statements.py

```diff
@@ -26,7 +26,8 @@ def nits(self):
       # Python 3 interpreter will raise SyntaxError upon reading a print statement.
       # So, this module cannot be meaningfully used when ran with a Python 3 interpreter.
       return
-    for print_stmt in self.iter_ast_types(ast.Print):
+    # MyPy says this is unreachable when run with Python 3
```

Comment:
Hm, unclear what the future of this custom `checker` code is. Toolchain does not have use for it now that we have Flake8. It will also be trivial for us to add Pylint.

Unless users have a strong use case towards this, I've been biasing towards trying to remove as much custom code (and config) as possible. Imo, Pants is best a wrapper around other tools like MyPy and Flake8. If we do care enough about this custom linter to port to V2, I'd like to see it moved to a new dedicated repo and be released as a linter that functions independently of Pants, i.e. decouple it from Pants. Then, Pants can wrap the linter just like it wraps Flake8.

Reply:
Things in `contrib/` are luckily very easy to separate out like that!